docs/source/tutorials/getting-started/lifecycling.rst

```diff
@@ -101,7 +101,7 @@ the associated entitlements.
   :start-after: -- CLAIM_EVENT_BEGIN
   :end-before: -- CLAIM_EVENT_END
 
-As a side-effect of claiming the effect the presented holding is exchanged for a holding of the new
+As a side-effect of claiming the effect, the presented holding is exchanged for a holding of the new
```

Comment:
```suggestion
As a side-effect of claiming the effect, the presented holding is exchanged for a holding of the new
```

This is not entirely correct:
- the holding is exchanged once the instructions generated by claiming an effect are settled

a Holder can claim an effect twice, but can settle only once.

Fine to fix this in a separate PR

Reply:
👍